icetea/ContractInvoker.js

```diff
@@ -98,7 +100,15 @@ class ContractInvoker {
     const src = Buffer.from(tx.data.src, 'base64')
     const deployedBy = tx.from
     const vm = getRunner(mode)
-    const compiledSrc = vm.compile(src)
+    let compiledSrc = vm.compile(src)
+
+    // metering
+    if (mode === ContractMode.WASM) {
```

Comment:
Why don't we do this inside vm/WasmRunner.compile, because it is designed to do this work.

Reply:
ok, got it